src/test/python/complete_winml_tutorial.py

```diff
@@ -29,42 +29,42 @@ def sync_wrapper(*args, **kwds):
 
     return async_wrapper if asyncio.iscoroutinefunction(fun) else sync_wrapper
 
-import pyrt.windows.ai.machinelearning as winml 
+import winrt.windows.ai.machinelearning as winml 
 import os
 
 @timed_op
 def load_model(model_path):
-    return winml.LearningModel.LoadFromFilePath(os.fspath(model_path))
+    return winml.LearningModel.load_from_file_path(os.fspath(model_path))
```

Comment:
Here the class appears to have the right casing but the method again has a different casing. Seems rather confusing.

Reply:
Again, python convention for members is lower_snake_case as per #201

Reply:
Sure, I get that Python has that convention just like C++ has the convention of using lower_case for types and members. But that does not mean that it is a requirement. It's going to get rather difficult to avoid having a growing list of exceptions as you deal with various APIs that you you need to special case. It is also lossy in the sense that you cannot readily roundtrip the name back to the original. C++/WinRT takes the approach that while C++ libraries tend to use lower_case, API authors choose what to name their APIs and how they are cased. This is far less problematic. Anyway, just an observation.